src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -868,6 +868,19 @@ protected function _BuildFilterSection()
 
             $filterContent .= '<div class="form-group mr-2 typeahead-relative">';
 
+            $filterContent .= '<script>
+                function resetPagingAndSearch(event) {
+                    if (event !== undefined && typeof event.key !== "undefined") {
+                        if (event.key !== "Enter") {
+                            return;
+                        }
```

Comment:
If it is the enter key, `event.preventDefault()` should probably be called, just to prevent race-conditions, where the browsers default behaviour is to submit the form while we also do that programatically.

Reply:
ok

Reply:
Alright.